Add `Catch French Dual Tournament 2` wiki article (#7881)
ppy/osu-wiki: wiki/Tournaments/CFC/CFDT_2/en.md

---
tags:
  - CFC
  - CFDT
  - CFDT2
  - CFDT 2
---

# Catch French Dual Tournament 2

![CFDT 2 logo](img/logo.png)

> The **Catch French Dual Tournament 2** (***CFDT 2***) is a French double-elimination 2v2 osu!catch tournament hosted by ::{ flag=FR }:: [h-e-c-a123](https://osu.ppy.sh/users/3163012) and ::{ flag=FR }:: [Laffey Gaming](https://osu.ppy.sh/users/5122949). Two players will form a team with first rank range is between #1 and #500 and the second rank range is rank #501 or below. It is the second instalment of the Catch French Dual Tournament and part of the Catch French Cup series.

Reviewer: ```suggestion
The **Catch French Dual Tournament 2** (***CFDT 2***) is a French double-elimination 2v2 osu!catch tournament hosted by ::{ flag=FR }:: [h-e-c-a123](https://osu.ppy.sh/users/3163012) and ::{ flag=FR }:: [Laffey Gaming](https://osu.ppy.sh/users/5122949). Teams consist of two players, where the first player is ranked between #1 and #500, and the latter is ranked #501 or below. It is the second instalment of the Catch French Dual Tournament and part of the Catch French Cup series.
```

## Tournament schedule

| Event | Timestamp |
| --: | :-- |
| Registration phase | 2022-07-03/2022-07-24 |
| Screening phase | 2022-07-25/2022-08-07 |
| Live drawings | 2022-08-14 (21:00 UTC+2) |
| Qualifiers | 2022-08-16/2022-08-21 |
| Round of 16 | 2022-08-23/2022-08-24 |
| Quarterfinals | 2022-08-30/2022-08-31 |
| Semifinals | 2022-09-03/2022-09-04 |
| Finals | 2022-09-10/2022-09-11 |
| Grand Finals | 2022-09-17/2022-09-18 |

## Prizes

| Placing | Prize(s) |
| :-: | :-- |
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | Unique profile badge, 2 months of osu!supporter |

> | ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 1 months of osu!supporter |

Reviewer: ```suggestion
| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 1 month of osu!supporter |
```

## Organisation

> The Catch French Dual Tournament 2 was run by various community members.

Reviewer: was? the rest of the article is in present tense
